task01.rb

```diff
@@ -0,0 +1,10 @@
+#1. Write a script that returns true if a string contains comparison, e.g. "<23", ">45".
+
+def task01(str)
+  str.gsub!(/>>/, "")
+  str.gsub!(/<</, "")
+  str.gsub!(/ /, "")
+  str.gsub!(/([<,>,=])(\d+)(\+)/) {">=" + $2}
```

Comment:
@MaraEfimova You should not try to "fix" the input, we are only accepting spaces (when they are not in the comparison operator). In case of invalid input - we should throw an error.